workflow/environments/README.md

```diff
@@ -22,33 +22,62 @@ A user specific environment of workflow, qcore, IMCalc, Empirical Engine and Pre
 can be created as follows:  
 1) Go to an existing copy of workflow (eg. can be your home) - it is a good idea to update this copy
 2) Navigate to ".../slurm_gm_workflow/workflow/environments/org/nesi"
-3) Load the correct version of Python module. (** IMPORTANT **) (eg. for Python 3.9, try `module load cray-python cray-hdf5-parallel/1.12.2.3`)
+3) Load the correct version of Python module. (** IMPORTANT **). For example, Python 3.9, try the following.
+   ```bash
+   module load cray-python cray-hdf5-parallel/1.12.2.3
+   ```
 4) Run   in /nesi/project/nesi00213/Environments/
     ```bash
-    ./create_env_maui.sh environment_name config_to_use
+    ./create_env_maui.sh ENV_NAME ./env_config.json
     ```
     where a default config (env_config.json) is located in the same directory, which should
-    work without requiring any changes.  
-    Note: The environment is installed into /nesi/project/nesi00213/Environments/
+    work without requiring any changes. 
+    Note: The environment is created at /nesi/project/nesi00213/Environments/ENV_NAME
     so if an environment with the same already exists the script will exit.
 
-5) Check that the script ran to completion without any errors, 
-apart from the IM_calculation setup warning and the pip qcore error.
+5) Check that the script ran to completion and gave any error messages.
```

Comment:
and gave no error messages?

Reply:
We expect some error messages. :) 
Added what error messages are expected and how to handle them in the same paragraph. But I agree that the grammar can be a bit more refined :)